CMakeLists.txt

```diff
@@ -48,8 +48,8 @@ ENDIF()
 FIND_PROGRAM(CMAKE_AR_TMP NAMES ar)
 
 # detect top level projects
-FIND_PACKAGE( Yajl )
```

Comment:
Yeah, that would do the trick as well. ;)

My patch would have led to bundled yajl and elektra being used when someone hadn't set -DUSE_SYSTEM_ instead.